code/game/objects/items/ego_weapons/zayin.dm

```diff
@@ -19,32 +81,115 @@
 	attack_verb_simple = list("slices", "slashes", "stabs")
 	hitsound = 'sound/weapons/bladeslice.ogg'
 
-/obj/item/ego_weapon/wingbeat
+	matching_armor = /obj/item/clothing/suit/armor/ego_gear/little_alice
+	use_message = "You use little alice to share snacks!"
+	use_sound = "sound/items/eatfood.ogg"
+	ability_cooldown_time = 60 SECONDS
+
+/obj/item/ego_weapon/support/little_alice/Pulse(mob/living/carbon/human/user)
+	var/list/foodoptions = list(/obj/item/food/cookie, /obj/item/food/cookie/sugar/pbird)
+	for(var/mob/living/carbon/human/L in livinginview(5, user))
+		if((!ishuman(L)) || L.stat == DEAD || L == user)
+			continue
+		if(L.nutrition > NUTRITION_LEVEL_WELL_FED)
+			continue
+		to_chat(L, "<span class='warning'>[user] gives you a snack!</span>")
+		var/gift = pick(foodoptions)
+		new gift(get_turf(L))
+
+/obj/item/ego_weapon/support/wingbeat
 	name = "wingbeat"
 	desc = "If NAME can show that they are competent, then they may be able to draw Fairy Festival’s attention.."
+	special = "Use this weapon in your hand when wearing matching armor to activate a special ability."
 	icon_state = "wingbeat"
 	force = 14
 	damtype = RED_DAMAGE
 	armortype = RED_DAMAGE
 	attack_verb_continuous = list("smacks", "strikes", "beats")
 	attack_verb_simple = list("smack", "strike", "beat")
 
+	matching_armor = /obj/item/clothing/suit/armor/ego_gear/wingbeat
+	pulse_enable_toggle = TRUE
+	use_message = "You use wingbeat to emit healing pulses!"
+	use_sound = "sound/abnormalities/fairyfestival/fairylaugh.ogg"
+	var/pulse_healing = -0.5
+
+/obj/item/ego_weapon/support/wingbeat/Pulse(mob/living/carbon/human/user, count)
+	..()
+	for(var/mob/living/carbon/human/L in livinginview(4, user))
+		if(L.stat == DEAD || L == user || L.is_working) //no self-healing
+			continue
+		L.adjustBruteLoss(pulse_healing)
+		to_chat(L, "<span class='nicegreen'>Fairies come from [user] to heal your wounds.</span>")
+
 /obj/item/ego_weapon/change
 	name = "change"
 	desc = "A hammer made with the desire to change anything"
+	special = "Attack a friendly human while wearing matching armor to activate this weapon's special ability"
 	icon_state = "change"
 	force = 14
 	damtype = RED_DAMAGE
 	armortype = RED_DAMAGE
 	attack_verb_continuous = list("slams", "strikes", "smashes")
 	attack_verb_simple = list("slam", "strike", "smash")
 
-/obj/item/ego_weapon/doze
+/obj/item/ego_weapon/change/attack(mob/living/M, mob/living/user)
+	if(!ishuman(M) || M == user)
+		..()
+		return
+	var/obj/item/clothing/suit/armor/ego_gear/change/C = user.get_item_by_slot(ITEM_SLOT_OCLOTHING)
+	if(!istype(C))
+		..()
+		return
+	var/mob/living/carbon/human/HT = M
+	if(HT.is_working)
+		to_chat(user,"<span class='notice'>You cannot defend others from responsibility!</span>")
+		return
+	if(user.faction_check_mob(HT))
+		user.playsound_local(get_turf(user), 'sound/abnormalities/change/change_end.ogg', 25, 0)
```

Comment:
Should probably be just a normal play sound, no? So people nearby, including the person getting hit, hear what's happening. May also wanna call the attack animation as well for visual indication.